packages/integration-tests/src/EntityManager.factories.test.ts

```diff
@@ -179,4 +183,22 @@ describe("EntityManager.factories", () => {
     expect(a1.firstName).toEqual("a1");
     expect(a1.lastName).toBeUndefined();
   });
+
+  it("should default children to empty array if created bottom-up", async () => {
```

Comment:
Forgot that we had a file dedicated to factory tests, so moved old test + new test here.